ci-operator/step-registry/ipi/cleanworkspace/kubevirt/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
```

Comment:
i think kubevirt/cleanworkspace would be a better start because imo something like this won't be prevalent enough to be this high level...

also maybe you use the post steps to make sure the cleanup happens always, and fall back to dptp run cleaner to cleanup all remaining things, instead of creating a cleanup pre step.

Reply:
This is a workaround because of a bug that I opened to dptp
https://issues.redhat.com/browse/DPTP-1884
I can't use the post steps for being sure that the cluster was destroyed...